src/vs/base/browser/ui/splitview/paneview.css

```diff
@@ -52,6 +52,14 @@
 	margin-left: auto;
 }
 
+/* {{SQL CARBON EDIT}} Bring back styles for action labels - ours aren't sized correctly without these */
```

Comment:
Maybe we should make our own css that brings in all these customizations so that we don't risk losing stuff in a merge again...we can just have some top-level component import it (like common-icons.css)

Reply:
agree, that moving these under sql makes sense.  I deleted these styles intentionally since they were causing issues with the terminal tree control, and when removing I didn't notice this side-effect.  It would be good to update the comment about which of our icons won't be sized correctly.